src/ng/compile.js

```diff
@@ -1508,6 +1508,14 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
       safeAddClass($element, isolated ? 'ng-isolate-scope' : 'ng-scope');
     } : noop;
 
+    compile.$createComment = function(directiveName, comment) {
```

Comment:
if it is `$$createComment` then for sure we should not use it in `ngMessagesInclude`